ui/app/components/task-sub-row.js

```diff
@@ -0,0 +1,76 @@
+import Ember from 'ember';
```

Comment:
All of this is largely ported from `task-row`, but updated to a glimmer component.